src/app/(dev)/playground/components/UpdateDevUserForm.tsx

```diff
@@ -91,7 +91,7 @@ const UpdateDevUserFrom = () => {
     <Card className="p-6 max-w-xlg mx-auto">
     <Form {...form}>
       <form
-        action={() => form.handleSubmit(handleSubmission)()}
+        action={form.handleSubmit(handleSubmission) as unknown as string}
```

Comment:
@Mubashar1009 why we need to use type assertion?

Reply:
fixed